source/crazyEights.py

```diff
@@ -0,0 +1,19 @@
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+
+from source import CardGames
+
```

Comment:
This is something I was trying to do with my code, but I couldn't quite get it to work. It looks really good tho! Nice and clean